lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -383,6 +393,12 @@ type GetByronWallet = "byron-wallets"
 type ListByronWallets = "byron-wallets"
     :> Get '[JSON] [ApiByronWallet]
 
+-- | https://input-output-hk.github.io/cardano-wallet/api/#operation/forceResyncByron
+type ForceResyncByronWallet = "byron-wallets"
+    :> Capture "walletId" (ApiT WalletId)
+    :> "utxos"
```

Comment:
the same question as above . why not `forceResync`

Reply:
After second thought - `DELETE /wallet/{walletID}/utxos` starts to make sense.... but was expecting that it would be `POST /wallet/{walletID}/resync` ....

Reply:
Well, it's more to stay close to a REST philosophy and intuition. In REST, endpoints are not "actions" or "commands", instead, they identify **resources**. Operations we can do on resources are defined by the HTTP verbs (create = POST, replace = PUT, read = GET etc ...). 

So, what would be the "resync" resource exactly :thinking: ? It doesn't really fit in the REST philosophy. Now, if we think of what the operation actually does: it removes all the UTxOs and rewind the wallet back to its original genesis block. To stick with the REST approach, I though of two possible ways:

- This one, using `DELETE /wallets/{walletId}/utxos`
- Or, thinking of it in terms of the slot: `PATCH /wallets/{walletId}/tip` with a body that contains a slot id (epoch no + slot no). 

I think the second approach is a little more intuitive, though, it suppose that the `tip` can be set to _anything_. The wallet backend might still enforce that the given tip argument is necessarily `(0, 0)` though. Perhaps that's a cleaner way of approaching this?

cc @piotr-iohk

Reply:
yeah,  resource `state` (instead of utxos) along with DELETE would be the most intuitive (for me). Indeed, it is good to stick to ```resources + VERB``` and hence to REST philosophy

Reply:
We also have `GET /wallets/{walletId}/statistics/utxos` which now I think could be just `GET /wallets/{walletId}/utxos` since we have `DELETE /wallets/{walletId}/utxos`...

Reply:
Okay, I've opted for the following in the end. 

```
PUT /wallets/{walletId}/tip
```

This requires a `{ "slotNumber": ..., "epochNumber:" ... }` as argument, but will reject anything that is different from the genesis point (0, 0). I think it's a bit clearer / more intuitive.

Reply:
Why to expose `{ "slotNumber": ..., "epochNumber:" ... }` since one cannot put anything other than (0,0)?